cfgov/jinja2/v1/_layouts/content-base-main-first.html

```diff
@@ -1,12 +1,22 @@
 {% extends 'content-base.html' %}
 
 {% block pre_content scoped -%}
-  {% if breadcrumb_items | length > 0 %}
-      <div class="content_wrapper">
-      {%- import 'breadcrumbs.html' as breadcrumbs -%}
-      {{ breadcrumbs.render(breadcrumb_items, 'breadcrumbs__main-first') }}
-      </div>
-  {% endif %}
+    {% if page %}
```

Comment:
I'd actually prefer to leave it like that because it makes removing the non-wagtail code easier. Instead of mixing it in multiple places, it separates concerns to a single block for each.

I don't have much of an issue with refactoring it like you've suggested if you feel strongly about it.